src/pages/Homepage/Homepage.jsx

```diff
@@ -23,11 +27,23 @@ const Homepage = () => {
             </p>
           </section>
           <section className=" homepage-left-action">
-            <button className="btn btn-primary btn-homepage">
-              <Link to="/mynotes"> Get Started</Link>
-            </button>
+            {!getUser() ? (
+              <>
+                <Link to="/login" className="btn btn-primary btn-homepage">
+                  Get Started
+                </Link>
 
-            <p className="cta-text">Register</p>
+                <Link to="/register" className="cta-text">
+                  Register
+                </Link>
+              </>
+            ) : (
+              <>
+                <Link to="/mynotes" className="btn btn-primary btn-homepage">
+                  Your Notes
+                </Link>
+              </>
```

Comment:
Empty brackets seems like extra tag here you can remove them

Reply:
Navbar was a great catch ! will definitely do it in future PR while refactoring!